packages/block-library/src/navigation/index.php

```diff
@@ -85,6 +85,41 @@ function block_core_navigation_build_css_font_sizes( $attributes ) {
 	return $font_sizes;
 }
 
+/**
+ * Renders a Navigation Block derived from data from the theme_location assigned
+ * via the block attribute 'location'.
+ *
+ * If the theme doesn't explicity support 'block-nav-menus' or no location was provided
+ * as a block attribute then an empty string is returned.
+ *
+ * @param  array $attributes Navigation block attributes.
+ * @return string HTML markup of a generated Navigation Block.
+ */
+function get_classic_navigation_elements( $attributes ) {
+
+	if ( ! current_theme_supports( 'block-nav-menus' ) ) {
+		return '';
+	}
```

Comment:
What is the role of checking for this here? The expected use of this supports is kind of the opposite, as here the result is rendering a classic menu, while the supports is meant to enable rendering of a block menu.

Reply:
The purpose isn't to render a classic menu, the purpose is to render a BLOCK menu with a classic data source.  That flag ensures that what comes out of the call to render the menu is navigation block markup.  Ideally it would be preferred to not require that theme supports and simply (in this situation) ALWAYS render block markup.  But this was the cleanest path to done at the moment.